Description: Remove tests that require internet connectivity.
Author: Felix Geyer <fgeyer@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=%23706517+ 
Forwarded: no
Last-Update: 2013-12-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- ruby-mini-magick-3.8.1.orig/spec/lib/mini_magick/image_spec.rb
+++ ruby-mini-magick-3.8.1/spec/lib/mini_magick/image_spec.rb
@@ -130,19 +130,19 @@ describe MiniMagick::Image do
       }.to_not raise_error
     end
 
-    it 'loads remote image' do
-      image = described_class.open('http://upload.wikimedia.org/wikipedia/en/b/bc/Wiki.png')
-      expect(image).to be_valid
-      image.destroy!
-    end
-
-    it 'loads remote image with complex url' do
-      image = described_class.open(
-        'http://a0.twimg.com/a/1296609216/images/fronts/logo_withbird_home.png?extra=foo&plus=bar'
-      )
-      expect(image).to be_valid
-      image.destroy!
-    end
+#     it 'loads remote image' do
+#       image = described_class.open('http://upload.wikimedia.org/wikipedia/en/b/bc/Wiki.png')
+#       expect(image).to be_valid
+#       image.destroy!
+#     end
+# 
+#     it 'loads remote image with complex url' do
+#       image = described_class.open(
+#         'http://a0.twimg.com/a/1296609216/images/fronts/logo_withbird_home.png?extra=foo&plus=bar'
+#       )
+#       expect(image).to be_valid
+#       image.destroy!
+#     end
 
     it 'reformats an image with a given extension' do
       expect {
